api/easymock/pom.xml

```diff
@@ -1,14 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
     <parent>
         <artifactId>powermock-api</artifactId>
         <groupId>org.powermock</groupId>
         <version>1.6.2-SNAPSHOT</version>
     </parent>
-    <modelVersion>4.0.0</modelVersion>
     <artifactId>powermock-api-easymock</artifactId>
-    <name>powermock-api-easymock</name>
-    <version>1.6.2-SNAPSHOT</version>
+    <name>${project.artifactId}</name>
```

Comment:
Hi René,

I'm just wondering if there is any difference - other than being more explicit - between using "project.artifactId" as "name" and not adding the "name" element. Doesn't maven use the "artifactId" as name when name is not set?